operations/locustfile.py

```diff
@@ -1,5 +1,6 @@
 from locust import HttpUser, task, events
 from locust.runners import MasterRunner
+import urllib.parse
```

Comment:
This is nitpicking, and no need to change it in this PR as we don't have much python code in our repo that we're maintaining and I don't think is necessary. So just as FYI. According to PEP8 (Python's style guide):
> Imports should be grouped in the following order:
>     Standard library imports.
>     Related third party imports.
>     Local application/library specific imports.

So, `import urllib.parse` should come first as it's a standard library; and `locust` is a third-party